app/src/main/java/com/example/nsc_events/screen/AddEvent.kt

```diff
@@ -18,24 +24,49 @@ import androidx.compose.material3.TopAppBar
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
-import androidx.compose.ui.Alignment
+
 import androidx.compose.ui.Modifier
+
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.unit.dp
 import androidx.navigation.NavHostController
 import com.example.nsc_events.Routes
 import com.example.nsc_events.data.Event
+import com.example.nsc_events.data.SocialMedia
+import java.util.Calendar
+import java.util.Date
+
+
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun AddEventPage(navController: NavHostController) {
-    var eventName by remember { mutableStateOf("") }
+    var eventTitle by remember { mutableStateOf("") }
     var eventDescription by remember { mutableStateOf("") }
-    var eventDate by remember { mutableStateOf("") }
-
+    var eventCategory by remember { mutableStateOf("") }
+    var eventDate by remember { mutableStateOf(Calendar.getInstance().time) }
+    var eventStartDate by remember { mutableStateOf(Calendar.getInstance().time) }
+    var eventEndDate by remember { mutableStateOf(Calendar.getInstance().time) }
```

Comment:
I believe startDate and endDate should use String and be relabeled to startTime and endTime to match the name and type in our schema https://github.com/SeattleColleges/nsc-events-nestjs/blob/main/src/activity/schemas/activity.schema.ts